### ISSUE TITLE
F005-SREBP diagram: Arcs from SREBP2 are not SBGN-compliant

### ISSUE THREAD

**Issue by ibalaur** [metabolismregulation/metabolismregulation.github.io#4]
SBGN validation for the arcs from SREBP2 returns error. To be re-checked in initial F005-SREBP.graphml file.
Input yEd file:
<img width="483" alt="InputF005-SREBP" src="https://user-images.githubusercontent.com/16227615/55631891-5c436e00-57b9-11e9-9a1b-a15efe9c4c22.png">
Output SBGN file (with validation):
<img width="489" alt="OutputF005-SREBP" src="https://user-images.githubusercontent.com/16227615/55631890-5c436e00-57b9-11e9-88a4-96f41057c3f0.png">

**Comment by amazein** [comment]
Yes, this is expected. It is not fully compliant SBGN. The connection to a submap handles differently in SBGN, via terminals, with no arrows... Let's think what to do about this, maybe simply remove.
